### ISSUE TITLE
[Adaptive Cards - ExpenseReport.Json]: 'Remove comment' button is not accessible by the keyboard in comment edit section under 'Amount' in 'Expense Approval' card .

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7266]
https://microsoft.visualstudio.com/OS/_workitems/edit/35352759

**Comment by vagpt** [comment]
Closing this issue as per the discussion with internal team as there are alternate ways are available to remove the text.
